mdoc/Mono.Documentation/Updater/Formatters/CSharpFullMemberFormatter.cs

```diff
@@ -484,33 +485,27 @@ private StringBuilder AppendParameter (StringBuilder buf, ParameterDefinition pa
 
         protected override string GetPropertyDeclaration (PropertyDefinition property)
         {
-            MethodDefinition method;
-
-            string get_visible = null;
-            if ((method = property.GetMethod) != null &&
-                    (DocUtils.IsExplicitlyImplemented (method) ||
-                     (!method.IsPrivate && !method.IsAssembly && !method.IsFamilyAndAssembly)))
-                get_visible = AppendVisibility (new StringBuilder (), method).ToString ();
-            string set_visible = null;
-            if ((method = property.SetMethod) != null &&
-                    (DocUtils.IsExplicitlyImplemented (method) ||
-                     (!method.IsPrivate && !method.IsAssembly && !method.IsFamilyAndAssembly)))
-                set_visible = AppendVisibility (new StringBuilder (), method).ToString ();
-
-            if ((set_visible == null) && (get_visible == null))
+            string getVisible = null;
```

Comment:
There doesn't really seem to have been a reason to change the `get_visible` variable names, and indeed, their usage further down in the code was not changed. Please revert this change, as it just adds noise to the PR/Diff without any real change to functionality.